source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/update/summary/__snapshots__/PropertyManagementUpdateContainer.test.tsx.snap

```diff
@@ -0,0 +1,10 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PropertyManagementUpdateContainer component renders as expected 1`] = `
+<DocumentFragment>
```

Comment:
Might need to wait for the page to load here.

Reply:
Will actually remove this snapshot as it is for the container test that renders an empty view which is why it shows like that. The view does have an snapshot test